tools/dockerfile/grpc_clang_format/clang_format_all_the_things.sh

```diff
@@ -44,7 +44,7 @@ fi
 
 if [ "$TEST" == "" ]
 then
-  echo $files | xargs -P $CPU_COUNT -n 10 $CLANG_FORMAT -i
+  echo $files | xargs -P $CPU_COUNT -n 1 $CLANG_FORMAT -i
```

Comment:
Why this change?

Reply:
if clang-format crashes this gives us the ability to examine the stack and also clang-format the rest of the files.